extensions/interactions/ItemSelectionInput/ItemSelectionInput.py

```diff
@@ -28,14 +28,23 @@ class ItemSelectionInput(base.BaseInteraction):
     answer_type = 'SetOfHtmlString'
 
     _customization_arg_specs = [{
-        'name': 'selection_count',
-        'description': 'Number of selections the student can make',
+        'name': 'max_allowable_selection_count',
+        'description': 'Maximum number of selections the student may make, '
+                       'please note that this is not the correct number of answers.',
         'schema': {
             'type': 'int'
         },
         'default_value': 1,
     }, {
-        'name': 'items',
+        'name': 'min_allowable_selection_count',
+        'description': 'Minimum number of selections the student may make, '
+                       'please note that this is not the correct number of answers.',
+        'schema': {
+            'type': 'int'
```

Comment:
made the validator to at least 1, because 0 selections isn't reasonable (perhaps we can field opinion on that). I'm thinking most creators will do a "None of the above" selection.